src/FHIRConverterAPI/Models/UserFacingException.cs

```diff
@@ -0,0 +1,18 @@
+using System.Net;
+
+public class UserFacingException: Exception
```

Comment:
could you add some doc string on this and the other models with a sentence or two on what they're for?